projects/backend/src/main/java/backend/service/UserService.java

```diff
@@ -0,0 +1,21 @@
+package backend.service;
+
+import backend.domain.User;
+
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+
+@Service
+public interface UserService {
+  void update(User user);
+
+  void add(User user);
+
+  void delete(User user);
+
+  User get(String userId);
+
```

Comment:
It's okay, but right now I realized that all ```Services``` that we had implemented offer ```CRUD``` operations. Maybe it would be nice to create a common generic interface to avoid code duplication, or see if spiring offers a ready-to-go interaces/classes